media-api/conf/routes

```diff
@@ -35,6 +35,7 @@ GET     /usage/quotas/:id                             controllers.UsageControlle
 # Management
 GET     /management/healthcheck                       com.gu.mediaservice.lib.management.ElasticSearchHealthCheck.healthCheck
 GET     /management/manifest                          com.gu.mediaservice.lib.management.ManagementWithPermissions.manifest
+GET     /management/imageCounts                       com.gu.mediaservice.lib.management.ElasticSearchHealthCheck.imageCounts
```

Comment:
Could we split the additional route into another PR? This is because the order of deployment isn't completely deterministic; if the lambda was to ship first it'll get a 404 until media-api had deployed too.

Reply:
[This has been moved here](https://github.com/guardian/grid/pull/2735)